Userland/Libraries/LibTest/Macros.h

```diff
@@ -24,8 +24,8 @@ namespace Test {
 TestResult current_test_result();
 void set_current_test_result(TestResult);
 
-RandomnessSource& randomness_source();
-void set_randomness_source(RandomnessSource);
+Randomized::RandomnessSource& randomness_source();
+void set_randomness_source(Randomized::RandomnessSource);
```

Comment:
Was this change meant to go here, or should it be in the commit that added these declarations?

Reply:
Moved it to the correct commit.